grader_host/index.js

```diff
@@ -406,8 +411,8 @@ function runJob(info, callback) {
   const globalJobTimeoutId = setTimeout(() => {
```

Comment:
Rename this variable to `jobTimeoutId` for consistency with the fact that we aren't using `globalJobTimeout` anymore?

Reply:
Done in 0ec0f34.